### ISSUE TITLE
[Research] Tabbed Nav – Keybindings 

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#2462]
**User Story:**
As Spark we want Tab and Shift-Tab behavior to comply with WAI-ARIA best practices 

**Notes:**
Requestor: Bob
WAI-ARIA best practices -> https://www.w3.org/TR/wai-aria-practices/#tabpanel

**AC:**
Research how keybindings should function.
- Tab and Shift-Tab behavior will comply with WAI-ARIA best practices
- UP and LEFT will go to previous tab?
- DOWN and RIGHT will go to next tab?
- HOME will go to first tab?
- END will go to last tab?
- Tabs can't be selected if disabled.(Angular fix #1176)
- none of the actions will scroll the page?
- Left/Right arrows will on function when component renders horizontally?
- Only if a tab has focus (not tab content)

**Comment by guern1kn** [comment]
For horizontal tablist:
- Right Arrow - Moves focus to the next tab. If focus is on the last tab element, moves focus to the first tab.
- Left Arrow - Moves focus to the previous tab. If focus is on the first tab element, moves focus to the last tab.
- Up Arrow - Scrolls the page.
- Down Arrow - Scrolls the page. 

For vertical tablist:
- Down Arrow - Moves focus to the next tab. If focus is on the last tab element, moves focus to the first tab.
- Up Arrow - Moves focus to the previous tab. If focus is on the first tab element, moves focus to the last tab.

Vertical or Horizontal:
- HOME - Moves focus to the first tab.
- END - Moves focus to the last tab.
- SPACE or ENTER - Activates the tab if it was not activated automatically on focus.
- A Tab cannot receive focus if it is DISABLED.
- The keyboard events will only change focus if a tab has focus (not the tabpanel or content inside has focus)

Sent this to Olya on Monday, Feb 24 for review. Will update this info based on her feedback.
